src/test/java/org/prebid/server/bidder/adf/AdfBidderTest.java

```diff
@@ -55,10 +57,34 @@ public void makeHttpRequestsShouldReturnExpectedBidRequest() {
         final BidRequest expectedRequest = bidRequest.toBuilder()
                 .imp(singletonList(bidRequest.getImp().get(0).toBuilder().tagid("12345").build()))
                 .build();
+
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1)
+                .extracting(HttpRequest::getPayload)
+                .flatExtracting(BidRequest::getImp)
+                .containsExactly(expectedRequest.getImp().toArray(new Imp[0]));
+    }
+
+    @Test
+    public void makeHttpRequestsShouldReturnRequestWithPriceTypeExt() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(
+                identity(),
+                extAdfBuilder -> extAdfBuilder
+                        .ext(jacksonMapper.fillExtension(ExtRequest.empty(),
+                                mapper.createObjectNode().set("pt", TextNode.valueOf("gross")))));
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = adfBidder.makeHttpRequests(bidRequest);
+
+        // then
         assertThat(result.getErrors()).isEmpty();
         assertThat(result.getValue()).hasSize(1)
-                .extracting(httpRequest -> mapper.readValue(httpRequest.getBody(), BidRequest.class))
-                .containsExactly(expectedRequest);
+                .extracting(HttpRequest::getPayload)
+                .extracting(BidRequest::getExt)
+                .containsOnly(jacksonMapper.fillExtension(
+                        ExtRequest.empty(), mapper.createObjectNode()
+                                .set("pt", TextNode.valueOf("gross"))));
     }
```

Comment:
This test does not follow the logic of the bidder. You need to add the `priceType` to the `imp.ext.priceType` and check that the adapter will put this value into `request.ext.pt`.